web/modules/custom/ecc_cct/config/schema/ecc_cct.schema.yml

```diff
@@ -0,0 +1,8 @@
+# Schema for the configuration files of the ECC Cookie Consent Tracker module.
+ecc_cct.settings:
+  type: config_object
+  label: 'ECC Cookie Consent Tracker settings'
+  mapping:
+    enable:
+      type: string
```

Comment:
Why not boolean?

Reply:
The exact why is lost in the mists of history, but I'm guessing it's an artifact of `drush generate:module`. It really doesn't matter one way or the other. It's stored as a string, it's read as a string, it works. I don't see it as having much impact either way?  I mean, we can change it, but it'll be change for change's sake.